apps/front/src/app/(auth)/login/page.tsx

```diff
@@ -0,0 +1,93 @@
+import React from 'react'
+import Image from 'next/image'
+import Logo from '../../../../public/assets/logo.png'
+import SignInPhoto from '../../../../public/assets/signInPhoto.png'
+import Link from 'next/link'
+import LoginForm from '@src/app/(auth)/login/loginForm/loginForm'
+import { InputAdornment, TextField } from '@mui/material'
+import ClipBoardCopy from '@src/components/ui/copyToClipboard'
+
+const Login = () => {
+    return (
+        <div className='min-h-screen text-white font-raleway flex justify-center items-center bg-black'>
+            <Image
+                src={Logo}
+                alt='Kanban logo'
+                className='absolute z-10 left-1 top-1'
+                width={60}
+                quality={100}
+            />
+            <div className='lg:w-2/3 m-5 bg-gradient-to-br from-[#00dffc] flex to-[#00ff82] items-center rounded-2xl border-[rgba(255,255,255,0)] [box-shadow:0px_0px_31px_9px_rgba(0,_0,_0,_1)] overflow-hidden relative'>
+                <div className='flex flex-col bg-[#141414] w-full lg:w-2/3 mr-auto rounded-2xl'>
+                    <div className='mx-5 my-10 lg:w-3/5'>
+                        <div className='text-4xl md:m-5 text-center m-3'>
+                            Login
+                        </div>
+                        <LoginForm />
+                        <div className='flex justify-center m-5'>
+                            Create new account
+                            <Link
+                                href='/register'
+                                className='text-green-300 underline ml-2'
+                            >
+                                Sign up
+                            </Link>
+                        </div>
+                        <div className='m-5 flex justify-around items-center gap-2'>
+                            <hr className='my-4 border-t border-white w-full' />
+                            or
+                            <hr className=' w-full my-4 border-t border-white' />
+                        </div>
+                        <div className='m-5 space-y-8'>
+                            <div className='text-center font-bold'>
+                                Try test account
```

Comment:
test account? Co to za feature?

Reply:
Nikomu się nie chce zakładać konta dlatego dałem textfieldy z loginem i hasłem do skopiowania aby można było szybko rzucić okiem co to za apka i jak działa

Reply:
raczej powinieneś mieć wejście bez logowania a dopiero po zalogowaniu użytkownik dostaję jakies feature'y np taski zapisują się w bazie, a nie tylko w przeglądarce. Takie konto poza tym jest podatne na to, że będą tam się znajdowały śmieci i będzie zapychało bazę. Inna sprawa, że musisz teraz na każdym środowisku utrzymywać konkretnie to konto na prodzie i lokalu zawsze musi być to konto i np jak ja teraz odpale projekt to muszę puścić jakąś query w sql aby mieć te testowe konto